stdlib/2and3/time.pyi

```diff
@@ -67,18 +64,15 @@ else:
         def __new__(cls, o: TimeTuple, _arg: Any = ...) -> struct_time: ...
 
 
-# ----- functions -----
-def asctime(t: Union[TimeTuple, struct_time, None] = ...) -> str: ...  # return current time
+def asctime(t: Optional[Union[TimeTuple, struct_time]] = ...) -> str: ...
 def clock() -> float: ...
-def ctime(secs: Union[float, None] = ...) -> str: ...  # return current time
-def gmtime(secs: Union[float, None] = ...) -> struct_time: ...  # return current time
-def localtime(secs: Union[float, None] = ...) -> struct_time: ...  # return current time
+def ctime(secs: Optional[float] = ...) -> str: ...
+def gmtime(secs: Optional[float] = ...) -> struct_time: ...
+def localtime(secs: Optional[float] = ...) -> struct_time: ...
 def mktime(t: Union[TimeTuple, struct_time]) -> float: ...
-def sleep(secs: Union[int, float]) -> None: ...
-def strftime(format: str,
-             t: Union[TimeTuple, struct_time, None] = ...) -> str: ...  # return current time
-def strptime(string: str,
-             format: str = ...) -> struct_time: ...
+def sleep(secs: float) -> None: ...
+def strftime(format: str, t: Optional[Union[TimeTuple, struct_time]] = ...) -> str: ...
```

Comment:
The `t` argument is not actually Optional (`time.strftime('%y', None)` throws an error).

Reply:
Same for asctime() actually. I am changing it to @overloads.

Reply:
Why do you need to? I think `def strftime(format: str, t: Union[TimeTuple, struct_time] = ...)` is fine.

Reply:
Because `strftime("%y")` and `asctime()` are legal calls (using the current time). Or am I overlooking something?

Reply:
That would be allowed by the code I gave above.

Reply:
Ah yes, of course, the default value.